yahoofantasy/resources/league.py

```diff
@@ -28,6 +28,21 @@ def teams(self, persist_ttl=DEFAULT_TTL):
             teams.append(t)
         return teams
 
+    def players(self, persist_ttl=DEFAULT_TTL):
+        logger.debug("Looking up players")
+        START = 0
+        COUNT = 25
+        data = self.ctx._load_or_fetch(f"players.{self.id}.{START}", f"players;count={COUNT};start={START}", league=self.id)
+        players = []
+        while "player" in data["fantasy_content"]["league"]["players"]:
+            for player in data["fantasy_content"]["league"]["players"]["player"]:
+                p = Player(self)
+                from_response_object(p, player)
+                players.append(p)
+            START += COUNT
+            data = self.ctx._load_or_fetch(f"players.{self.id}.{START}", f"players;count={COUNT};start={START}", league=self.id)
```

Comment:
to make a better and faster improvement, these request may be sent simultaneously in the future.